fem/fespace.hpp

```diff
@@ -775,11 +775,16 @@ class FiniteElementSpace
 
    /** @brief Mark degrees of freedom associated with boundary elements with
        the specified boundary attributes (marked in 'bdr_attr_is_ess').
+
        For spaces with 'vdim' > 1, the 'component' parameter can be used
-       to restricts the marked vDOFs to the specified component. */
+       to restricts the marked vDOFs to the specified component.
+       If overwrite is set to false then values in ess_vdofs are preserved
+       and not reset. However, the assumption here is that ess_vdofs is set to
+       the correct size already.*/
    virtual void GetEssentialVDofs(const Array<int> &bdr_attr_is_ess,
                                   Array<int> &ess_vdofs,
-                                  int component = -1) const;
+                                  int component = -1,
+                                  bool overwrite = true) const;
```

Comment:
Yes, it's just away to avoid having `ess_vdofs` being recreated every time this function is called and the need for another `Array` to keep track of the accumulated values. If you think improved documentation would help here to make that more clear I can do that.